readme-sync/v0/senseml-reference/8000 - concepts/1800 - jsonlogic.md

```diff
@@ -48,25 +49,23 @@ Returns a boolean to indicate if the specified value exists.
 
 ```json
 {
-    "exists": [
-        JsonLogic
-    ]
+    "exists": JsonLogic
 }
 ```
 
-Most commonly used with the JsonLogic `var`  operation to test that an output value isn't null. The  `var` operation retrieves extracted field values using field `id` keys. 
+Most commonly used with the JsonLogic `var` operation to test that an output value isn't null. Returns true if the value being evaluated is neither null nor undefined. Can take an array (where it will check the first item only) or a single value (e.g., `{ "exists": [{ "var": "some_field" }] }` or `{ "exists": { "var": "some_field" } }`).
 
 ### Examples
 
 See [Validating extractions](doc:validate-extractions#examples).
 
 ## Flatten
 
-Takes as input an array that can contain nested arrays, and returns a single-level array populated with the same values.  This operation is similar to the built-in JsonLogic [merge](https://jsonlogic.com/operations.html#merge) operation, except that it's recursive to any depth. 
```

Comment:
removed this note because this is not actually much like merge after all, even though we might have started thinking about it based on how merge works (merge takes multiple arrays and makes them one, flatten takes one array and makes it flat)